flake.nix

```diff
@@ -206,8 +208,10 @@
             fix (final: recursiveUpdate
               (defaultConfig final)
               (if isFunction extraConfig then extraConfig final else extraConfig));
-          nodeDbVol = "node-db";
-          nodeIpcVol = "node-ipc";
+          # Network subdirectory in cardano-configurations repo provided above
+          network = "testnet";
```

Comment:
Move this line to `defaultConfig`, you wouldn't be able to override it here. Then you need to add `inherit (config) network` or replace all of the occurrences with the qualified name (i.e. `config.network`)